src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -800,30 +800,41 @@ export default class GroupComparisonStore {
         }
     });
 
+    public readonly samplesVennPartition = remoteData({
+        await:()=>[
+            this._activeGroupsNotOverlapRemoved,
+            this.sampleSet,
+            this.samples
+        ],
+        invoke:()=>{
+            const sampleSet = this.sampleSet.result!;
+            return Promise.resolve(
+                partitionCases(
+                    this._activeGroupsNotOverlapRemoved.result!,
+                    (group)=>getSampleIdentifiers([group]),
+                    (id)=>sampleSet.get({ studyId: id.studyId, sampleId: id.sampleId })!.uniqueSampleKey,
```

Comment:
this is actually important - for ComplexKey data structures you have to make a specific object because it uses the entire object as a key. typescript doesnt ensure that -it only ensures the object meets an interface